.github/workflows/test.yaml

```diff
@@ -81,27 +79,20 @@ jobs:
     - name: Dump cluster info
       run: |
         kubectl cluster-info dump
-    - name: Setup needed resources
-      run: |
-        kubectl create namespace astarte-test
-        make install
     - name: Run e2e tests
-      run: go test -v -timeout 20m ./...
-      working-directory: "test/e2e${{ matrix.testSuite }}"
-      env:
-        USE_EXISTING_CLUSTER: "true"
+      run: make test-e2e ignore-not-found=true
     - name: Dump debug info
       if: failure()
       run: |
-        kubectl get pods -n astarte-test
-        kubectl get deployment -n astarte-test
-        kubectl get statefulset -n astarte-test
-        kubectl get secrets -n astarte-test
-        kubectl get configmaps -n astarte-test
-        kubectl get pvc -n astarte-test
-        kubectl describe astarte -n astarte-test || echo "Astarte already deleted"
-        kubectl logs -n astarte-test deployment/example-astarte-flow || echo "No logs"
-        kubectl logs -n astarte-test deployment/example-astarte-flow --previous || echo "No logs"
+        kubectl get pods
+        kubectl get deployment
+        kubectl get statefulset
+        kubectl get secrets
+        kubectl get configmaps
+        kubectl get pvc
+        kubectl describe astarte || echo "Astarte already deleted"
+        kubectl logs deployment/example-astarte-flow || echo "No logs"
+        kubectl logs deployment/example-astarte-flow --previous || echo "No logs"
     - name: Show pod states
       run: |
-        kubectl get pods -n astarte-test
+        kubectl get pods
```

Comment:
Shouldn't a namespace be specified here?

Reply:
E2e tests are performed by installing the Astarte CR in the default namespace. It's not advised in production, but it keeps things simple(r) for our tests.
I could update tests to install in a dedicated namespace. I need to think about it.